client-libraries/Bash/pantry-client.bash

```diff
@@ -0,0 +1,67 @@
+# Library of bash-functions to access https://getpantry.cloud/ API
+#
+# To avail the functions listed herein, source this script as
+#
+# source ./pantry-client.bash "Your_Pantry_ID_here"
+#
+# Tip: Put the above command inside ~/.bashrc to avoid sourcing manually everytime
+#
+# Author: Somajit Dey <dey.somajit@gmail.com> 2021
+
+export PANTRY_ID="${1:-"${PANTRY_ID}"}"
+
+[[ -z "${PANTRY_ID}" ]] && \
+echo 'Warning: PANTRY_ID not set
+Execute command: export PANTRY_ID="Your_Pantry_ID_here"'
```

Comment:
Nice!

Reply:
Thank you so much. I have squashed all my commits too.